modules/nf-core/happy/ftxpy/tests/main.nf.test

```diff
@@ -0,0 +1,122 @@
+nextflow_process {
+
+    name "Test Process HAPPY_FTXPY"
+    script "../main.nf"
+    process "HAPPY_FTXPY"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "happy"
+    tag "happy/ftxpy"
+
+    test("homo_sapiens - illumina - mutect") {
+        when {
+            process {
+                """
+                input[0] = [
+                        [ id:'test' ], // meta map
+                        "hcc.mutect.snv",
+                        file(params.test_data['homo_sapiens']['illumina']['test_test2_paired_mutect2_calls_vcf_gz'], checkIfExists: true),
+                        [],
+                        [],
+                        []
+                        ]
+                input[1] = [
+                        [ id:'test2' ], // meta map
+                        file(params.test_data['homo_sapiens']['genome']['genome_21_fasta'], checkIfExists: true)
+                        ]
+                input[2] = [
+                        [ id:'test3' ], // meta map
+                        file(params.test_data['homo_sapiens']['genome']['genome_21_fasta_fai'], checkIfExists: true)
+                        ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    process.out.features,
```

Comment:
This could just be process.out and cover both parts

Reply:
thanks!